frontend/packages/integration-tests-cypress/tests/storage/snapshot.spec.ts

```diff
@@ -0,0 +1,89 @@
+import { testName, checkErrors } from '../../support';
+import {
+  SnapshotDetails,
+  snapshotClassDropdown,
+  dropdownFirstItem,
+  pvcDropdown,
+} from '../../views/storage/snapshot';
+import { listPage, status } from '../../views/list-page';
+import { PVC, testerDeployment, SnapshotClass } from '../../mocks/snapshot';
+import { detailsPage } from '../../views/details-page';
+import { submitButton } from '../../views/form';
+
+const snapshotName = `${PVC.metadata.name}-snapshot`;
+
+if (Cypress.env('BRIDGE_AWS')) {
+  describe('Snapshot Tests', () => {
+    before(() => {
+      cy.login();
+      cy.createProject(testName);
+      cy.exec(`echo '${JSON.stringify(PVC)}' | oc apply -n ${testName} -f -`);
+      cy.exec(`echo '${JSON.stringify(testerDeployment)}' | oc apply -n ${testName} -f -`);
+      cy.exec(`echo '${JSON.stringify(SnapshotClass)}' | oc apply -f -`);
+      cy.clickNavLink(['Storage', 'Persistent Volume Claims']);
+      listPage.filter.byName(PVC.metadata.name);
+      // Let the PVC get bound
+      cy.contains(status, 'Bound');
+    });
+
+    afterEach(() => {
+      checkErrors();
+    });
+
+    after(() => {
+      cy.exec(`echo '${JSON.stringify(testerDeployment)}' | oc delete -n ${testName} -f -`);
+      cy.exec(`echo '${JSON.stringify(PVC)}' | oc delete -n ${testName} -f -`);
+      cy.exec(`echo '${JSON.stringify(SnapshotClass)}' | oc delete -f -`);
+      cy.deleteProject(testName);
+      cy.logout();
+    });
+
+    it('Creates Snapshot', () => {
+      cy.clickNavLink(['Volume Snapshots']);
+      listPage.clickCreateYAMLbutton();
+      cy.byTestID(pvcDropdown).click();
```

Comment:
Not sure why this needs to be in a variable, just as easy to write `pvc-dropdown`.